install-build-deps.sh

```diff
@@ -18,11 +18,4 @@ go install github.com/kevinburke/go-bindata/go-bindata
 go install github.com/vektra/mockery/cmd/mockery
 go install github.com/weaveworks/github-release
 go install golang.org/x/tools/cmd/stringer
-
-# TODO: metalinter is archived, we should switch to github.com/golangci/golangci-lint
-# Install metalinter
-# Managing all linters that gometalinter uses with dep is going to take
-# a lot of work, so we install all of those from the release tarball
-METALINTER_VERSION="3.0.0"
-curl --silent --location "https://github.com/alecthomas/gometalinter/releases/download/v${METALINTER_VERSION}/gometalinter-${METALINTER_VERSION}-${OSARCH}.tar.gz" \
-  | tar -x -z -C "${GOBIN}" --strip-components 1
+go install github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
There should be `go.sum` and `go,mod` entries for it also.

Reply:
@errordeveloper I suppose to run `make -f Makefile.docker update-build-image-manifest && make -f Makefile.docker push-build-image` as I made changes for this file. But it seems like i don't have a permission to push docker image.

Reply:
due to https://github.com/golangci/golangci-lint/issues/821, I don't think we can include `golangci-lint` in go.mod, go.sum and tools.go. Please advise

Reply:
@errordeveloper can you help with the above permission? Seems like the build failed as new docker image is not pushed.###

Reply:
@errordeveloper `golangci-lint` now can be installed with `go get`, so I have updated tools.go as your suggestion. Please help to take another look.